actors/builtin/miner/miner_actor.go

```diff
@@ -669,6 +669,13 @@ func (a Actor) ConfirmSectorProofsValid(rt Runtime, params *builtin.ConfirmSecto
 			// compute initial pledge
 			activation := rt.CurrEpoch()
 			duration := precommit.Info.Expiration - activation
+
+			// Avoid math errors if duration is too low somehow.
+			if duration <= 0 {
```

Comment:
I wonder if we should be checking min sector lifetime instead?

Reply:
Yes.
1. PreCommitSector should add the max seal duration to current epoch when invoking `validateExpiration()` ([from the issue](https://github.com/filecoin-project/specs-actors/issues/795#issuecomment-664100263))
2. Here we should check again that the min lifetime meets the bound (which we can assert is positive)